frontend/packages/dev-console/package.json

```
@@ -24,7 +24,8 @@
     },
     "exposedModules": {
       "icons": "src/utils/icons.tsx",
-      "catalog": "src/components/catalog"
+      "catalog": "src/components/catalog",
+      "jarFileUpload": "src/components/jar-file-upload/jar-file-upload-utils.ts"
```

Try to expose a module instead of just one component or utility. Create an index.ts file that can export more than just this specific utility in future. Like all file upload utilities and components can be exposed from this module.
```suggestion
      "fileUpload": "src/components/jar-file-upload/jar-file-upload-utils.ts"
```